src/products/iOS_products/updating_locales.md

```diff
@@ -0,0 +1,29 @@
+# Updating locales for specific projects
+
+These are the instructions to update locales on iOS mobile products, currently living within the [Firefox for iOS](https://github.com/mozilla-l10n/firefoxios-l10n), and in the [Focus for iOS](https://github.com/mozilla-l10n/focusios-l10n) projects.
+
+## Prerequisites to adding a locale
+
+First, you will have to check that the locale is available within the iOS system settings. iOS currently supports two tiers of locales, and a locale must be part of one of these two lists in order to ship on Mozilla products for iOS. Unlike Mozilla products for Android, there’s no locale switcher included in the app, so it’s possible to ship only locales that are supported by iOS.
+
+To tell if a locale is supported or not, take a look on an actual device, under the Language Settings of the most recent iOS version available. Tier one locales are the ones listed under “iPod/iPhone/iPad Language”, and Tier 2 locales appear under “Other Language” (or “Add Language” if there are already some languages under that list). If a locale doesn’t appear in any of these lists, then it is currently not supported by iOS.
+
+## Updating locales in Pontoon
+
+Once you’ve determined if the locale can be added you must go to Pontoon to update the list of current locales. If the locale is not [available](https://pontoon.mozilla.org/teams/) in Pontoon yet, [consult this document](../../tools/pontoon/adding_new_locale.md) for instructions on how to add it.
+
+In Pontoon, you can add and remove locales from the admin project page: [here](https://pontoon.mozilla.org/admin/projects/firefox-for-ios/) for Firefox for iOS, and [here](https://pontoon.mozilla.org/admin/projects/focus-for-ios/) for Focus for iOS. The project will be available after the next [Pontoon sync](https://pontoon.mozilla.org/sync/log/).
+
+### Adding locales
+
+Once you have added a new locale in Pontoon, there is nothing more to do in order to get the locale into Nightly builds. This should happen automatically within 24-48 hours. To check that the locale is indeed now set up in the l10n repository, you can check that its correspoding locale folder has been created [here](https://github.com/mozilla-l10n/firefoxios-l10n) for Firefox for iOS and [here](https://github.com/mozilla-l10n/focusios-l10n) for Focus for iOS.
+To check that the locale made in into the code repository, there is a script running (note that this method may not always be 100% reliable): [here](https://github.com/mozilla-l10n/firefoxios-l10n#locales-in-build) for Firefox for iOS and [here](https://github.com/mozilla-l10n/focusios-l10n#locales-in-build) for Focus for iOS. A more reliable method would be to look for the pull request that imports translations in the code repository, and verify that it created files for that new locale.
```

Comment:
A reminder that a simple line break doesn't do what you think it does (it's equivalent to a space). You can see it [here](https://github.com/mozilla-l10n/documentation/blob/6b7fdd7f8204d874d886cc5c42d0e8d377d5071c/src/products/iOS_products/updating_locales.md), where this section is a massive block of text.

Reply:
Thanks for catching that!